31Aug_MajorityElement.py

```diff
@@ -0,0 +1,33 @@
+# https://leetcode.com/problems/majority-element/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+#####################################################################################################################################################
+#
+# Given an array nums of size n, return the majority element.
+# The majority element is the element that appears more than ⌊n / 2⌋ times. You may assume that the majority element always exists in the array.
+#
+# Example 1:
+# Input: nums = [3,2,3]
+# Output: 3
+#
+# Example 2:
+# Input: nums = [2,2,1,1,1,2,2]
+# Output: 2
+#
+class Solution:
+    def majorityElement(self, nums: List[int]) -> int:
```

Comment:
This errors because `List` isn't imported

`NameError: name 'List' is not defined. Did you mean: 'list'?`

Reply:
There wont be explicit imports here, as these are handled at the backend. By default, every Leetcode problem has this statement:

```
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
```
    
We code on top of it.